.gitattributes

```diff
@@ -0,0 +1,97 @@
+* text=auto !eol svneol=native#text/plain
```

Comment:
I don't think we need this file checked in here, right?

Reply:
I'll be contributing said file back to ASIO upstream. I'm fairly sure Chris will accept that bit.

Reply:
https://github.com/chriskohlhoff/asio/pull/444

Reply:
Okay, we'll make sure to rebase when updating.